app/db/config/config.js

```diff
@@ -0,0 +1,28 @@
+require('dotenv').config()
+
+module.exports = {
+  'development': {
+    'username': process.env.PG_USERNAME,
```

Comment:
node-postgres (relied on by Sequelize) will look for PGUSER, PGPASSWORD, PGDATABASE, PGHOST, PGPORT environment variables if you don't supply these options to Sequelize.